Servers/auth/utils/SSOValidation.ts

```
@@ -0,0 +1,351 @@
+    return { isValid: errors.length === 0, errors };
+  }
+
+  /**
+   * Sanitize display name from SSO provider
+   */
+  static sanitizeDisplayName(name: string): string {
+    if (!name) return '';
+
+    return name
+      .trim()
+      .replace(/[<>{}[\]\\\/\x00-\x1f\x7f]/g, '') // Remove dangerous characters
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)